### PR TITLE
clean up code

### PR DESCRIPTION
**Pull request by mragab2000**

### CODE REVIEW

File: oop/chair.php [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-
+declare(strict_types=1);
 class Chair
 {
     public string $color;
     public float $price;
     public string $brand;
 
-    public function backPainTreatment($painType)
+    public function setting(string $bodyWeight)
```
Comment (mohamedragabdahab) [inline oop/chair.php]: please add missing return type

File: oop/chair.php [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     {
-        return $painType;
+        return $bodyWeight;
     }
 }
 $chair1 = new Chair;
```
Comment (mohamedragabdahab) [inline oop/chair.php:14]: add missing constructor brackets ()

File: oop/chair.php [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 $chair1->color = "brown";
 $chair1->price = 1500;
 $chair1->brand = "ikea";
-$chair1->backPainTreatment("spine issue");
\ No newline at end of file
+$chair1->setting("90kg");
```
Comment (mohamedragabdahab) [inline oop/chair.php:18]: add missing constructor brackets ()

File: oop/mug.php [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+declare(strict_types=1);
 class Mug
 {
     //(color - price - size)
     private string $color;
     public float $price;
     public float $size;
 
-    public function drinkingHotDrink($drinkType)
+    public function drinkingHotDrink(string $drinkType)
```
Comment (mohamedragabdahab) [inline oop/mug.php]: add missing return type

File: oop/pen.php [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
-	$pen1 = new pen();
-	$pen1->color = "red";
-	$pen1->price = "15";
-    $pen1->expirationDate = "1/jun/2024";
-	$pen1->write("Hello");
\ No newline at end of file
+declare(strict_types=1);
+class Pen
+{
+  //variables
```
Comment (mohamedragabdahab) [inline oop/pen.php]: delete un needed comment

File: oop/pen.php [hunk 6]
```diff
@@ -1,22 +1,22 @@
...
\ No newline at end of file
+declare(strict_types=1);
+class Pen
+{
+  //variables 
+  public string $color;
+  public float $price;
+  public string $expirationDate;
+
+  public function write(string $text)
```
Comment (mohamedragabdahab) [inline oop/pen.php]: missing return type

File: oop/pen.php [hunk 7]
```diff
@@ -1,22 +1,22 @@
...
+  {
+
+    return $text;
+  }
+
+}
+
+$pen1 = new pen();
+$pen1->color = "red";
+$pen1->price = "15";
```
Comment (mohamedragabdahab) [inline oop/pen.php]: price is float, remove quotas

File: oop/mug.php [hunk 8]
```diff
@@ -1,18 +1,18 @@
 <?php
-
+declare(strict_types=1);
 class Mug
 {
     //(color - price - size)
```
Comment (mohamedragabdahab) [inline oop/mug.php]: remove comment

File: oop/chair.php (hunk 1 above)
Comment (mragab2000) [reply oop/chair.php]: fixed

File: oop/chair.php (hunk 3 above)
Comment (mragab2000) [reply oop/chair.php:18]: fixed

File: oop/mug.php (hunk 4 above)
Comment (mragab2000) [reply oop/mug.php]: fixed

File: oop/pen.php (hunk 5 above)
Comment (mragab2000) [reply oop/pen.php]: done

File: oop/pen.php (hunk 6 above)
Comment (mragab2000) [reply oop/pen.php]: fixed

File: oop/pen.php (hunk 7 above)
Comment (mragab2000) [reply oop/pen.php]: fixed

**Comment by mohamedragabdahab** [comment]
@mragab2000 looks good to me, you can merge 👍

**Review by mohamedragabdahab — APPROVED** [review]

**Pull request merged**